src/client/app/actions/barReadings.ts

```diff
@@ -112,18 +113,20 @@ function fetchGroupBarReadings(groupIDs: number[], timeInterval: TimeInterval):
 	};
 }
 
-function fetchMeterCompareReadings(meterIDs: number[], timeInterval: TimeInterval): Thunk {
+function fetchMeterCompareReadings(meterIDs: number[], comparePeriod: ComparePeriod): Thunk {
```

Comment:
This isn't an issue with your code, but could you remove the unused `const stringifiedIDs = groupIDs.join(',');` on line 110 in `fetchGroupBarReadings`?